<launch>

  <!-- Start openpose_ros -->
  <node pkg="openpose_ros" type="openpose_ros_node" name="openpose_ros_node">
    <param name="image_topic"   	value="/kinect2_head/rgb_lowres/image" />
    <param name="output_topic"      value="/openpose_ros/human_list" />
  </node>
</launch>
